### ISSUE TITLE
unable to use iou = "0.1.0"

### ISSUE THREAD

**Issue by balajijinnah** [withoutboats/iou#21]
getting this following error

```
error[E0425]: cannot find function `rust_io_uring_cq_advance` in crate `sys`
  --> /home/schoolboy/.cargo/registry/src/github.com-1ecc6299db9ec823/iou-0.1.0/src/cqe.rs:95:18
   |
95 |             sys::rust_io_uring_cq_advance(self.ring.as_ptr(), 1);
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `io_uring_cq_advance`

```

**Comment by withoutboats** [comment]
Sorry, I must not have understood how cargo does version resolution for pre-releases. I'll release an 0.2.0 of iou soon that will fix this. For now try adding a dependency on `uring-sys = "=1.0.0-alpha"`.

**Comment by withoutboats** [comment]
I've released iou `0.2.0`. I recommend updating, because it contains several bug fixes. :)
